src/blinding.cpp

```diff
@@ -492,6 +544,21 @@ LIBSESSION_C_API bool session_blind25_key_pair(
     }
 }
 
+LIBSESSION_C_API bool session_blind_version_key_pair(
+        const unsigned char* ed25519_seckey,
+        unsigned char* blinded_pk_out,
+        unsigned char* blinded_sk_out) {
+    try {
+        auto result = session::blind_version_key_pair({ed25519_seckey, 64});
+        auto [b_pk, b_sk] = result;
```

Comment:
As written, this is going to make a copy of the pk and sk values.  Two ways around it:

Don't use the intermediate `result` and read them directly into b_pk/b_sk:
```suggestion
        auto [b_pk, b_sk] = session::blind_version_key_pair({ed25519_seckey, 64});
```
Or make b_pk, b_sk references into `result`:
```suggestion
        auto result = session::blind_version_key_pair({ed25519_seckey, 64});
        auto& [b_pk, b_sk] = result;
```

Reply:
Not sure why I did this - I guess I just copied it from `session_blind25_key_pair` and assumed it was needed... and it looks like I wrote those functions as well so no idea 😂 

I've updated all of them to avoid this now